### ISSUE TITLE
Mario Party 9 (NTSC-U) is not recognized as WII DISC

### ISSUE THREAD

**Issue by GoogleCodeExporter** [andersonx007/cfg-loader-mod#24]
```
What steps will reproduce the problem?
1. Put in Mario Party 9 using CFG 70 r22
2. Try to "install" MP9 in hard drive

What is the expected output? What do you see instead?
Error will tell: Disc is not a WII DISC
It should install game on hard-drive

What version of the product are you using? On what operating system?
CFG v70 r22 (WII)
4.3U
d2x v7

Please provide any additional information below.
A post I made on GBATEMP: 
http://gbatemp.net/topic/147638-configurable-usb-loader/page__view__findpost__p_
_4138330

Basically installing MP9 using the original CFG v70 (non modded) works, but it 
doesn't work with CFG v70 r22 (modded one)
```

Original issue reported on code.google.com by `krunchth...@gmail.com` on 12 Mar 2012 at 11:06

**Comment by GoogleCodeExporter** [comment]
```
Try the attached version.
```

Original comment by `softwiiv2` on 25 Mar 2012 at 9:36
- Changed state: **Started**

Attachments:
- [cfg70r23t1.dol](https://storage.googleapis.com/google-code-attachments/cfg-loader-mod/issue-24/comment-1/cfg70r23t1.dol)

**Comment by GoogleCodeExporter** [comment]
```
some problem, all the retail Wii Disc can not be recognized, but cfg v70 
non-mod does
hope next mod version will fix it.
```

Original comment by `wu_...@hotmail.com` on 26 Mar 2012 at 2:07

**Comment by GoogleCodeExporter** [comment]
```
Have you tried the cfg70r23t1.dol from my first post?
Have you tested alle mod versions?
Which version breaks the Wii Disc Support?
```

Original comment by `softwiiv2` on 26 Mar 2012 at 4:59

**Comment by GoogleCodeExporter** [comment]
```
I did not have time to test it yet, but it looks like ISSUE 17 is the same 
problem as this. http://code.google.com/p/cfg-loader-mod/issues/detail?id=17
```

Original comment by `krunchth...@gmail.com` on 27 Mar 2012 at 2:01

**Comment by GoogleCodeExporter** [comment]
```
this r23t1 works now, can rip disc as before, thanks for your great job
```

Original comment by `wu_...@hotmail.com` on 27 Mar 2012 at 10:49

**Comment by GoogleCodeExporter** [comment]
```
fixed in r24
```

Original comment by `softwiiv2` on 3 Apr 2012 at 4:27
- Changed state: **Fixed**
